### PR TITLE
Make `Meta/serenity.sh rebuild-world` work on macOS without requiring stuff from brew

### PR DESCRIPTION
**Pull request by nico**
With these small changes, having just ninja and cmake on the path is enough to get the system built.

### CODE REVIEW

**Comment by ADKaster** [comment]
Looks good but shellcheck is unhappy

**Comment by nico** [comment]
Shellcheck and I had a heart to heart and resolved our differences.

**Review by ADKaster — APPROVED** [review]

File: Toolchain/BuildCMake.sh [hunk 1]
```diff
@@ -22,6 +22,23 @@ fi
 
 [ -z "$MAKEJOBS" ] && MAKEJOBS=$($NPROC)
 
+check_sha() {
```
Comment (GMTA) [inline Toolchain/BuildCMake.sh:25]: The commit message for this change refers to "brew on PATH", but I think you mean "sha256sum on PATH"?

Comment (nico) [reply Toolchain/BuildCMake.sh:25]: sha256sum is installed via brew (via the coreutils package). I mean sha256sum, md5sum, `install -D` -- in short, stuff from brew.

**Pull request merged**